Gemfile

```diff
@@ -13,51 +13,21 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
   end
 end
 
-if RUBY_VERSION < '1.9.3'
-  gem 'rake', '< 11.0.0' # rake 11 requires Ruby 1.9.3 or later
-elsif RUBY_VERSION < '2.0.0'
-  gem 'rake', '< 12.0.0' # rake 12 requires Ruby 2.0.0 or later
-else
-  gem 'rake', '>= 12.3.3'
-end
-
 if ENV['DIFF_LCS_VERSION']
   gem 'diff-lcs', ENV['DIFF_LCS_VERSION']
 else
   gem 'diff-lcs', '~> 1.4', '>= 1.4.3'
 end
 
-if RUBY_VERSION < '2.3.0' && !!(RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/)
-  gem "childprocess", "< 1.0.0"
-elsif RUBY_VERSION < '2.3.0'
-  gem "childprocess", "< 3.0.0"
-else
-  gem "childprocess", ">= 3.0.0"
-end
+gem "childprocess", ">= 3.0.0"
+gem 'ffi', '~> 1.13.0'
```

Comment:
This is incorrect, it needs 1.12 until when theres no Rubygems usage?

Reply:
Again, I left this intact 😆 

![image](https://user-images.githubusercontent.com/6916/101819428-4bbecb80-3b36-11eb-8aa9-9b35980cafeb.png)

Will fix this across repos 👍